### PR TITLE
i18n(es): Update `styling.mdx` & `tina-cms.mdx` 

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Update `styling.mdx` & `tina-cms.mdx`

`styling.mdx` [`58bc79a`](https://github.com/withastro/docs/commit/58bc79a6a39f5c25902a642f6e9579e8a43d8b91)

`tina-cms.mdx` [`a250850`](https://github.com/withastro/docs/commit/a2508507e6e87195eee6dfdadb887292fe40bfde)

### CODE REVIEW

File: src/content/docs/es/guides/styling.mdx [hunk 1]
```diff
@@ -515,11 +515,11 @@ Svelte en Astro también funciona exactamente como se espera: [Svelte Styling Do
...
+Cualquier método de estilo en Astro está disponible para un [componente de diseño de Markdown](/es/core-concepts/layouts/#plantillas-de-markdownmdx), pero métodos diferentes tendrán efectos de estilo diferentes en tu página.
 
-Puede aplicar estilos globales a tu contenido de Markdown agregando [etiquetas `<style>`](#estilos-globales) y [hojas de estilo importadas](#estilos-externos) al la plantilla que envuelve el contenido de tu página. También puedes agregar [integraciones de CSS](#integraciones-css) incluyendo [Tailwind](/es/guides/integrations-guide/tailwind/).
+Puedes aplicar estilos globales a tu contenido de Markdown agregando [hojas de estilo importadas](#estilos-externos) al diseño que envuelve el contenido de tu página. También es posible estilizar tu Markdown con etiquetas [`<style is:global>`](#estilos-globales) en el componente de diseño. Ten en cuenta que cualquier estilo agregado está sujeto al [orden de cascada de Astro](#orden-de-cascada), por lo que debes revisar cuidadosamente tu página renderizada para asegurarte de que los estilos se apliquen como se pretende.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/styling.mdx:520]:
```suggestion
Puedes aplicar estilos globales a tu contenido de Markdown agregando [hojas de estilo importadas](#estilos-externos) a la plantilla que envuelve el contenido de tu página. También es posible estilizar tu Markdown con etiquetas [`<style is:global>`](#estilos-globales) en el componente de diseño. Ten en cuenta que cualquier estilo agregado está sujeto al [orden de cascada de Astro](#orden-de-cascada), por lo que debes revisar cuidadosamente tu página renderizada para asegurarte de que los estilos se apliquen como se pretende.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Buen trabajo @Waxer59 😁, te dejo una pequeña sugerencia.

**Comment by Waxer59** [comment]
Listo!, gracias por la revisión 🤠

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work peeps! 💜

**Pull request merged**